### DISCUSSION TITLE
Latest update 20220317 Error

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#849] [Fixed Issues]
Hi, 
Look a chmod problem during the update process 
Impossible de run the update.
<img width="554" alt="Capture d’écran 2022-03-17 à 08 03 41" src="https://user-images.githubusercontent.com/77485841/158755012-8c0d4c44-92c2-4f4e-b8d8-5fede2f4271c.png">

**Comment by ceejayemm** [comment]
Not getting syntax error any longer but now get:

Unexpected string in JSON at position 139

Chris

[rAudio.html.tar.gz](https://github.com/rern/rAudio-1/files/8283512/rAudio.html.tar.gz)
1

**Reply by rern** [reply]
Try:
`sed -i '/latest/ d' /srv/http/data/system/display`

**Reply by ceejayemm** [reply]
@rern 

Thanks that fixed it for me.

Chris

**Comment by hjfisch** [comment]
Thank`s , working now

**Comment by ceejayemm** [comment]
Still JSON error for me.

Chris

**Comment by ceejayemm** [comment]
Rern's reply in post 3 above was the answer for me.

Chris
